server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -75,6 +76,7 @@ public class ExtensionsOrchestrator implements ReportingService<PluginsAndModule
     public static final String REQUEST_OPENSEARCH_NAMED_WRITEABLE_REGISTRY = "internal:discovery/namedwriteableregistry";
     public static final String REQUEST_OPENSEARCH_PARSE_NAMED_WRITEABLE = "internal:discovery/parsenamedwriteable";
     public static final String REQUEST_REST_EXECUTE_ON_EXTENSION_ACTION = "internal:extensions/restexecuteonextensiontaction";
+    public static final String REQUEST_EXTENSION_AUTHORIZE_ACTION = "internal:extensions/authorize";
```

Comment:
what does `internal` mean here?

Reply:
I believe `internal` means that this actions is performed node -> node.

Reply:
But this action will be extension -> node, should there be a different convention?

Reply:
Actually nix that previous thought, lets just consider this 'external' We shouldn't be `extension` centric for security features